sys/aarch64/rootdev.c

```diff
@@ -290,15 +290,17 @@ static void rootdev_deactivate_resource(device_t *dev, res_type_t type,
   /* TODO: unmap mapped resources. */
 }
 
-/* clang-format off */
-static bus_driver_t rootdev_driver = {
+static struct rootdev_driver {
+  driver_t driver;
+  bus_methods_t bus;
+} rootdev_driver = {
```

Comment:
NIT:
imo, it is easier to read when definition and declarations are separated - I mean:

```
typedef struct  {
  driver_t driver;
  bus_methods_t bus;
} rootdev_driver_t;

static rootdev_driver_t rootdev_driver = {
```
or something